spyder/plugins/preferences/api.py

```diff
@@ -909,6 +754,24 @@ def restart(self):
         """Restart Spyder."""
         self.main.restart()
 
+    def add_tab(self, Widget):
+        widget = Widget(self)
+        if self.tabs is None:
+            self.tabs = QTabWidget()
+            layout = self.layout()
+            main_widget = QWidget()
+            main_widget.setLayout(layout)
+            self.tabs.addTab(self.create_tab(main_widget),
+                             _('General'))

```

Comment:
Why do you need to create this extra tab here?

Reply:
This happens when adding a new tab into a page that didn't have any other one